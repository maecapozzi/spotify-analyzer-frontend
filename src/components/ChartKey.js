import React from 'react'

const ChartKey = () => (
  <div>
    <p>Danceability: "A measure from 0.0 to 1.0 of "how suitable a track is for dancing."</p>
    <p>Acousticness: "A confidence measure from 0.0 to 1.0 of whether the track is acoustic."</p>
    <p>Energy: "A measure from 0.0 to 1.0 that represents a perceptual measure of intensity and activity."</p>
    <p>Liveness: "Detects the presence of an audience in the recording."</p>
    <p>Instrumentalness: "Predicts whether a track contains no vocals."</p>
    <p>Speechiness: "Speechiness detects the presence of spoken words in a track."</p>
    <p>Valence: "A measure from 0.0 to 1.0 describing the musical positiveness conveyed by a track."</p>
  </div>
)

export default ChartKey
