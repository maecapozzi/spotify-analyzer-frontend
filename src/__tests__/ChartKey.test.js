import React from 'react'
import { shallow } from 'enzyme'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-15'

import ChartKey from '../components/ChartKey'

configure({ adapter: new Adapter() })

test('it renders correctly', () => {
  const component = shallow(<ChartKey />)
  expect(component).toMatchSnapshot()
})
