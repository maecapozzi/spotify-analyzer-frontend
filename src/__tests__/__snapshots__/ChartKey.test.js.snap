// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ChartKey />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          Danceability: "A measure from 0.0 to 1.0 of "how suitable a track is for dancing."
</p>,
        <p>
          Acousticness: "A confidence measure from 0.0 to 1.0 of whether the track is acoustic."
</p>,
        <p>
          Energy: "A measure from 0.0 to 1.0 that represents a perceptual measure of intensity and activity."
</p>,
        <p>
          Liveness: "Detects the presence of an audience in the recording."
</p>,
        <p>
          Instrumentalness: "Predicts whether a track contains no vocals."
</p>,
        <p>
          Speechiness: "Speechiness detects the presence of spoken words in a track."
</p>,
        <p>
          Valence: "A measure from 0.0 to 1.0 describing the musical positiveness conveyed by a track."
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Danceability: \\"A measure from 0.0 to 1.0 of \\"how suitable a track is for dancing.\\"",
        },
        "ref": null,
        "rendered": "Danceability: \\"A measure from 0.0 to 1.0 of \\"how suitable a track is for dancing.\\"",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Acousticness: \\"A confidence measure from 0.0 to 1.0 of whether the track is acoustic.\\"",
        },
        "ref": null,
        "rendered": "Acousticness: \\"A confidence measure from 0.0 to 1.0 of whether the track is acoustic.\\"",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Energy: \\"A measure from 0.0 to 1.0 that represents a perceptual measure of intensity and activity.\\"",
        },
        "ref": null,
        "rendered": "Energy: \\"A measure from 0.0 to 1.0 that represents a perceptual measure of intensity and activity.\\"",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Liveness: \\"Detects the presence of an audience in the recording.\\"",
        },
        "ref": null,
        "rendered": "Liveness: \\"Detects the presence of an audience in the recording.\\"",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Instrumentalness: \\"Predicts whether a track contains no vocals.\\"",
        },
        "ref": null,
        "rendered": "Instrumentalness: \\"Predicts whether a track contains no vocals.\\"",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Speechiness: \\"Speechiness detects the presence of spoken words in a track.\\"",
        },
        "ref": null,
        "rendered": "Speechiness: \\"Speechiness detects the presence of spoken words in a track.\\"",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Valence: \\"A measure from 0.0 to 1.0 describing the musical positiveness conveyed by a track.\\"",
        },
        "ref": null,
        "rendered": "Valence: \\"A measure from 0.0 to 1.0 describing the musical positiveness conveyed by a track.\\"",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            Danceability: "A measure from 0.0 to 1.0 of "how suitable a track is for dancing."
</p>,
          <p>
            Acousticness: "A confidence measure from 0.0 to 1.0 of whether the track is acoustic."
</p>,
          <p>
            Energy: "A measure from 0.0 to 1.0 that represents a perceptual measure of intensity and activity."
</p>,
          <p>
            Liveness: "Detects the presence of an audience in the recording."
</p>,
          <p>
            Instrumentalness: "Predicts whether a track contains no vocals."
</p>,
          <p>
            Speechiness: "Speechiness detects the presence of spoken words in a track."
</p>,
          <p>
            Valence: "A measure from 0.0 to 1.0 describing the musical positiveness conveyed by a track."
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Danceability: \\"A measure from 0.0 to 1.0 of \\"how suitable a track is for dancing.\\"",
          },
          "ref": null,
          "rendered": "Danceability: \\"A measure from 0.0 to 1.0 of \\"how suitable a track is for dancing.\\"",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Acousticness: \\"A confidence measure from 0.0 to 1.0 of whether the track is acoustic.\\"",
          },
          "ref": null,
          "rendered": "Acousticness: \\"A confidence measure from 0.0 to 1.0 of whether the track is acoustic.\\"",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Energy: \\"A measure from 0.0 to 1.0 that represents a perceptual measure of intensity and activity.\\"",
          },
          "ref": null,
          "rendered": "Energy: \\"A measure from 0.0 to 1.0 that represents a perceptual measure of intensity and activity.\\"",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Liveness: \\"Detects the presence of an audience in the recording.\\"",
          },
          "ref": null,
          "rendered": "Liveness: \\"Detects the presence of an audience in the recording.\\"",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Instrumentalness: \\"Predicts whether a track contains no vocals.\\"",
          },
          "ref": null,
          "rendered": "Instrumentalness: \\"Predicts whether a track contains no vocals.\\"",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Speechiness: \\"Speechiness detects the presence of spoken words in a track.\\"",
          },
          "ref": null,
          "rendered": "Speechiness: \\"Speechiness detects the presence of spoken words in a track.\\"",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Valence: \\"A measure from 0.0 to 1.0 describing the musical positiveness conveyed by a track.\\"",
          },
          "ref": null,
          "rendered": "Valence: \\"A measure from 0.0 to 1.0 describing the musical positiveness conveyed by a track.\\"",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
